src/commands/codepush/deployment/list.ts

```diff
@@ -0,0 +1,30 @@
+import { AppCommand, CommandArgs, CommandResult, help, failure, ErrorCodes, success, getCurrentApp } from "../../../util/commandline";
+import { out } from "../../../util/interaction";
+import { DefaultApp } from "../../../util/profile";
+import { inspect } from "util";
+import { MobileCenterClient, models, clientRequest } from "../../../util/apis";
+const _ = require("lodash");
+
+const debug = require("debug")("mobile-center-cli:commands:codepush:deployments:list");
+
+@help("List the deployments associated with an app")
```

Comment:
I think this should be changed to "List the deployments associated with your current app" so the user knows its applicable to their current application.